### ISSUE TITLE
Incorrect parsing of | inside code blocks in GFM tables

### ISSUE THREAD

**Issue by randomdude999** [raphlinus/pulldown-cmark#821]
[example 200 of the GFM spec](https://github.github.com/gfm/#example-200):
```markdown
| f\|oo  |
| ------ |
| b `\|` az |
| b **\|** im |
```
gets parsed to:
```html
<table><thead><tr><th>f|oo</th></tr></thead><tbody>
<tr><td>b <code>\|</code> az</td></tr>
<tr><td>b <strong>|</strong> im</td></tr>
</tbody></table>
```
note that the | inside the code block still has a backslash in front of it, which it shouldn't according to the GFM spec.

**Comment by ehuss** [comment]
I believe this has already been fixed via #699, and should be in the next release.

**Comment by randomdude999** [comment]
right, it is, sorry!
